### DISCUSSION TITLE
Test Discussion

### DISCUSSION THREAD

**Discussion by derrickmehaffy** [strapi/strapi#5838] [Legacy Topics]
I am just testing the discussions to see what kind of possibilities exists with them.

Please ignore me 😄

**Comment by techguydave** [comment]
It's like an issue, except it's not! 🚀

**Comment by abdonrd** [comment]
Looks really great! 👏

**Comment by robertpiosik** [comment]
I would suggest to reduce amount of places where support is provided. Slack is useless really because is not indexed by search engines, unanswered questions are lost quickly and so on.

**Reply by techguydave** [reply]
Actually, that's a really good point about SEO. A lot of repeat questions seem to come up because people don't search Slack, likely due to the nature of it being a real-time chat app. They search Google, don't find an answer, then go there.

Having support here would provide the benefit of the search results, as well as just minimizing the amount of places you need to look/post.

**Reply by derrickmehaffy** [reply]
@robertpiosik that is exactly what the Strapi team (and myself) are in the process of doing. I have just had my PR recently merged that more or less removes all instances of Stack Overflow from Strapi's code, documentation, and misc other areas. Likewise per [Jim's message](https://spectrum.chat/strapi/general/migrating-to-github-discussions~bb9ed525-6ba9-4793-9b7d-04f4252632a8) on the Spectrum Chat:  The Spectrum is being killed as well.

As far as "provided support" the end goal over the next few weeks and months is simply put:
- Github Discussions for questions and other misc stuff (while I was entertaining the idea of starting up forums, that has been put on hold while we beta test these discussions)
- Github issues for Bug reports and Feature requests (FRs will get moved to Product Board if the Strapi team plan to take them on, else community members can pick them up to work on PRs or take them and build plugins ect)
- Slack for real time convos and what not
- The various targeted repos for special cases like Docker, SDK, Strapi-Awesome, Community Articles/Tutorials/ect

On a side note, thanks for the feedback :wink:

**Reply by robertpiosik** [reply]
@derrickmehaffy That's great! I think we can update [this documentation page](https://strapi.io/documentation/3.0.0-beta.x/getting-started/troubleshooting.html#technical-support) to reflect new direction strapi is taking on this field. I can take care of it.

**Reply by derrickmehaffy** [reply]
It's already been updated, the new docs just haven't been deployed as it's a holiday weekend in France (where most of the Strapi team is located)

https://github.com/strapi/strapi/pull/5844

**Reply by robertpiosik** [reply]
Ahh, haven't checked it, awesome:)
